stdlib/public/stubs/LibcShims.cpp

```diff
@@ -308,3 +318,72 @@ swift::_stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound) {
   std::uniform_int_distribution<__swift_uint32_t> RandomUniform(0, upper_bound);
   return RandomUniform(getGlobalMT19937());
 }
+
+// _stdlib_random
+//
+// Should the implementation of this function add a new platform/change for a
+// platform, make sure to also update the stdlib documentation regarding
+// platform implementation of this function.
+// This can be found at: stdlib/public/core/Random.swift
+
+#if defined(_WIN32) && !defined(__CYGWIN__)
+#error TODO: Test _stdlib_random on Windows
```

Comment:
Is this going to break the build for Win32 platforms?

Reply:
Yes and no, this does break builds if they're compiling the stdlib. I can probably substitute this in with a warning directive instead.

Reply:
CC @compnerd